### DISCUSSION TITLE
Evolve in ASP .Net WebForm project

### DISCUSSION THREAD

**Discussion by apbertoletti** [lecaillon/Evolve#236] [Q&A] [answered]
Hi guys,

Does anybody know a use example of evolve-db in a ASP .Net WebForm project?

Is there any issue or limit to use evolve-db with this environment?

Thanks

**Comment by lecaillon** [comment, marked as answer]
Hi, you can use Evolve in a WebForm project as long as it supports **.NETStandard 2.0** (.NET Framework > 4.6.1)

**Reply by apbertoletti** [reply]
Thanks @lecaillon !
